symfony/templates/workqueue/consents.html.twig

```diff
@@ -41,14 +41,7 @@
             {% include 'partials/workqueue-consents-advance-filters.html.twig' %}
         </div>
         <div class="col-md-9">
-            <div class="text-right form-inline">
-                <div class="page-drop-down">
-                    Page <select name="pageDropDown" class="form-control input-sm"></select>&nbsp;of&nbsp;<span class="total-pages"></span>
-                </div>
-                {% if canExport %}
-                    <button data-href="{{ path('workqueue_export', params) }}" class="btn btn-xs btn-default export"><i class="fa fa-download" aria-hidden="true"></i> Export</button>
-                {% endif %}
-            </div>
+            {% include 'partials/workqueue-drop-down-navigation-and-eport-button.html.twig' %}
```

Comment:
Typo here (eport). But can we also shorten the file name? Maybe something like `workqueue-page-export.html.twig`?